docs/relnotes.md

```diff
@@ -18,6 +18,7 @@ This is the list of changes to Taipy releases as they were published.
    `Task.get_parents()^`, or `Pipeline.get_parents()^`, or by passing the data node entity, task entity or pipeline
    entity to the function `taipy.get_parents()^`.
    - The data node caching mechanism now also takes into account the date and time of the last modification of a file caused by either Taipy execution or an external factor. This behavior is limited to file-based data nodes: CSV, Excel, JSON, and pickle data nodes only.
+   - Added support for modin as the new exposed type.
```

Comment:
External link to https://github.com/modin-project/modin?

Reply:
I took the libery to use the home link https://modin.readthedocs.io/en/stable/ instead of their github link

Reply:
I don't really mind.